src/containers/Profile/Profile.tsx

```diff
@@ -1,42 +1,48 @@
-import * as React from 'react'
+import React, { useCallback, useEffect, useMemo } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
 import { Profile as BaseProfile } from 'decentraland-ui/dist/components/Profile/Profile'
+import { loadProfileRequest } from '../../modules/profile/actions'
+import { getData as getProfiles } from '../../modules/profile/selectors'
 import { Props } from './Profile.types'
 
-export default class Profile extends React.PureComponent<Props> {
-  static defaultProps = {
-    inline: true
-  }
+const Profile = function<T extends React.ElementType>(props: Props<T>) {
+  const { address, debounce, inline = true } = props
+  const profiles = useSelector(getProfiles)
+  const dispatch = useDispatch()
 
-  timeout: NodeJS.Timeout | null = null
+  let timeout: NodeJS.Timeout | null = null
```

Comment:
As we're using a functional component, the contents of timeout won't be there after a render re-trigger. To make it persistent, we can use `useRef`:
```suggestion
  let timeout: number | null = useRef(null)
```